presto-hive/src/test/java/com/facebook/presto/hive/TestDomainTranslator.java

```diff
@@ -125,6 +141,16 @@ private RowExpression dereference(RowExpression base, int field)
         return specialForm(DEREFERENCE, fieldType, ImmutableList.of(base, new ConstantExpression((long) field, INTEGER)));
     }
 
+    private RowExpression isNull(RowExpression expression)
+    {
+        return specialForm(IS_NULL, BOOLEAN, expression);
+    }
+
+    private RowExpression not(RowExpression expression)
+    {
+        return call("not", metadata.getFunctionManager().lookupFunction("not", fromTypes(BOOLEAN)), BOOLEAN, expression);
```

Comment:
"not" function is already available in `StandardFunctionResolution`. But I don't mind having it here given it's a test.

Reply:
@highker Like this?

`return call("not", new FunctionResolution(metadata.getFunctionManager()).notFunction(), BOOLEAN, expression);`

Reply:
@highker Like this?

`return call("not", new FunctionResolution(metadata.getFunctionManager()).notFunction(), BOOLEAN, expression);`